Modules/_threadmodule.c

```diff
@@ -1184,13 +1184,13 @@ thread_PyThread_start_new_thread(PyObject *self, PyObject *fargs)
         return NULL;
     }
 
-    struct bootstate *boot = PyMem_NEW(struct bootstate, 1);
+    struct bootstate *boot = PyMem_RawMalloc(sizeof(struct bootstate));
```

Comment:
Please add a comment to explain why PyMem_RawMalloc() is used instead of PyMem_Malloc(). Mention 
gh-109795.